frontend/src/pages/Common/Password.js

```diff
@@ -30,6 +30,7 @@ const Password = ({ password, storePassword, failed, nextBestAction, id, ...prop
     <div style={styles.container}>
       <PasswordIcon style={styles.icon} />
       <TextField
+        data-test="password-textfield"
```

Comment:
Could be problematic if you have mutliple password components on the screen
```suggestion
        data-test={props["data-test"] || "password-textfield"}
```

Reply:
done, also refactored `<Password>` component to be able to re-use it in the dialog